src/js/main.js

```diff
@@ -313,7 +316,7 @@ function startProcess() {
                         TABS.firmware_flasher.initialize(content_ready);
                         break;
                     case 'help':
-                        TABS.help.initialize(content_ready);
+                        import('./tabs/help').then(({ help }) => help.initialize(content_ready));
```

Comment:
Idea is to move the rest of the tabs to this usage. This way every tab and it's dependencies will be migrated one by one. Eventually leaving single `main.js` as entry point and removing the global variables.

Reply:
I've tried few different approaches. All of them are hard, because there was no clear separation on how to do it incrementally. This approach with `TABS` being the separation point is the most straightforwards.

we could definitely change the way tabs are exported and handled. That will be the next step. Didn't wan't to change a that because there are so many tabs already, and testing/reviewing a lot of changes will be hard enough already, there are a lot of other small things which have to be resolved. Like circular dependencies and random global variables.

Reply:
The approach to do the conversion step-by-step in this way seems reasonable.

What is the reason you chose to import the tabs here, instead of importing all of them at the top of the file, and just initialising them here?